ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/util/DbDispatcher.scala

```diff
@@ -22,7 +22,7 @@ trait DbDispatcher extends AutoCloseable {
     * The isolation level by default is the one defined in the JDBC driver, it can be however overriden per query on
     * the Connection. See further details at: https://docs.oracle.com/cd/E19830-01/819-4721/beamv/index.html
     * */
-  def executeSql[T](sql: Connection => T): Future[T]
+  def executeSql[T](description: String)(sql: Connection => T): Future[T]
```

Comment:
If we are planning to access this only if trace logging is enabled, it could make sense to pass this by name.

Reply:
Yes 👍

Reply:
Addressed in 310fa9d95.